### PR TITLE
Android/issue #10 create kitchen configuration screens

### PR DESCRIPTION
**Pull request by jgnt32**

### CODE REVIEW

File: app/src/main/java/com/weezlabs/drugreminder/fragments/WifiLocationFragment.java [hunk 1]
```diff
@@ -16,7 +20,17 @@
  */
 public class WifiLocationFragment extends Fragment implements View.OnClickListener {
 
+    public static final String PARENT_LOCATION_ID = "parent_location_id";
+
+    public static final int CAFETERIA = 1;
+    public static final int CUSTOM = 2;
+    private  final int KITCHEN = 0;
```
Comment (sunnyVolkova) [inline app/src/main/java/com/weezlabs/drugreminder/fragments/WifiLocationFragment.java]: Should be static?

File: app/src/main/java/com/weezlabs/drugreminder/fragments/WifiLocationFragment.java [hunk 2]
```diff
@@ -41,14 +55,45 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
...
+        editTextCustomPlaceName_ = (EditText) view.findViewById(R.id.edit_text_point_name);
+        spinner_ = (Spinner) view.findViewById(R.id.spinner_type);
+        ArrayAdapter<CharSequence> adapter = ArrayAdapter.createFromResource(getContext(), R.array.add_wifi_place_items, android.R.layout.simple_spinner_item);
+        adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+        spinner_.setAdapter(adapter);
+        spinner_.setSelection(0);
+        spinner_.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+                if (position == 2) {
```
Comment (sunnyVolkova) [inline app/src/main/java/com/weezlabs/drugreminder/fragments/WifiLocationFragment.java]: Later in code you compare with CUSTOM  constant, why not use it here?

File: app/src/main/java/com/weezlabs/drugreminder/fragments/WifiLocationFragment.java (hunk 1 above)
Comment (jgnt32) [inline app/src/main/java/com/weezlabs/drugreminder/fragments/WifiLocationFragment.java]: I use if for setting arguments. Maybe you can suggest better variant?

File: app/src/main/java/com/weezlabs/drugreminder/activities/HomeActivity.java [hunk 3]
```diff
@@ -111,7 +111,10 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
 		switch (requestCode){
 			case TUTORIAL_REQUEST_CODE:
-				startActivity(AddDrugReminderActivity.createIntent(this));
+				if (resultCode == RESULT_OK) {
```
Comment (sunnyVolkova) [inline app/src/main/java/com/weezlabs/drugreminder/activities/HomeActivity.java]: I think it is not clear way to inform about users choise. So probably better way is to move this logic to TutorialActivity

**Pull request merged**